src/workloads/scale/LLTInsert.yml

```diff
@@ -25,22 +25,31 @@ GlobalDefaults:
   # Loader Config.
   LoadThreads: &LoadThreads 4
   LoadBatchSize: &LoadBatchSize 1000
+
+  # In-memory
+  InitialDocumentCount: &InitialNumDocs 10000000
   SecondaryDocumentCount: &SecondaryNumDocs 10000000
+
+  # Not In-memory
+  #InitialDocumentCount: &InitialNumDocs 49000000
```

Comment:
Is out of memory future work, thus the commented code? If so, consider saying so.